### PR TITLE
Followup edits to PR #1053

### PR DESCRIPTION
**Pull request by bfallonf**
Nothing major. Just some edits.

If there's no problems, some one can hit merge.

@tpoitras @tnguyen-rh @ahardin-rh @adellape

### CODE REVIEW

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -792,16 +792,20 @@ specific profile with appropriate permissions.
...
-default */Common* partition.  Custom administrative partitions can be used
-to secure *F5 BIG-IP®* environments so that OpenShift specific
-configuration stored in *F5 BIG-IP®* system objects resides within a
-logical container.  This allows administrators to define access control
-policies on that specific administrative partition. To learn more about
-administrative partitions, please refer to the link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos_management_guide_10_0_0/tmos_partitions.html[*F5 BIG-IP®* documentation].
+custom *F5 BIG-IP®* administrative partition, instead of the default */Common*
+partition. You can use custom administrative partitions to secure *F5 BIG-IP®*
+environments. This means that an OpenShift-specific configuration stored in *F5
+BIG-IP®* system objects reside within a logical container, allowing
```
Comment (tpoitras) [inline install_config/install/deploy_router.adoc]: you might not want a line break in the middle of this term?

File: install_config/install/deploy_router.adoc [hunk 2]
```diff
@@ -792,16 +792,20 @@ specific profile with appropriate permissions.
...
+administrators to define access control policies on that specific administrative
+partition. 
+
+See the
+link:https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos_management_guide_10_0_0/tmos_partitions.html[*F5
+BIG-IP®* documentation] for more information about administrative partitions.
```
Comment (tpoitras) [inline install_config/install/deploy_router.adoc]: same here as above

**Comment by tpoitras** [comment]
nit-picked your line breaks

ignore at your own peril ;)

**Comment by bfallonf** [comment]
@tpoitras fixed

**Comment by adellape** [comment]
Looks like no revhistory needed.

**Pull request merged**